plugins/wazuh-core/public/services/query-manager/search-context.ts

```diff
@@ -0,0 +1,83 @@
+import { IndexPattern } from 'src/plugins/data/public';
+import {
+  TFilter,
+  IFilterManagerService,
+  IQueryService,
+  ISearchContext,
+  ISearchContextConfig,
+  QueryResult,
+} from './types';
+import { FilterManagerService } from './filter-manager-service';
+import { SearchParams } from './search-service';
+import { QueryService } from './query-service';
+
+export type QuerySearchParams = Omit<
+  SearchParams,
+  'indexPatternID' | 'filters'
+>;
```

Comment:
You could use OmitStrict (plugins/wazuh-core/common/types.ts) instead of Omit


```suggestion
export type QuerySearchParams = OmitStrict<
  SearchParams,
  'indexPatternID' | 'filters'
>;
```